graduation-project-tips.md

```diff
@@ -0,0 +1,93 @@
+# Ten Things they don't tell you about your graduation project
+
+In this document, I try to list the most important tips and tricks (in my opinion) in the hopes of helping future CCEians by sharing my experience with the graduation project and how to get the most out of it (hopefully).
+
+## Tip #1: Team Comes First :two_men_holding_hands:
+
+The team is the **most important factor** of any project, a team determines how the project starts, how the project ends and pretty much everything in between. 
+You must be comfortable to be yourself with your team and feel among friends, if this condition is not achieved, there will be fights and fights is not good for work.
```

Comment:
No need to be so negative I think. This is a bit extreme. Let's maybe advise people to be more professional and learn to work with people with a different mindset than theirs, but to expect less chemistry when working with people they haven't worked before.